packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4477,6 +4478,9 @@ class Semantics extends SingleChildRenderObjectWidget {
   /// all other tabs are unselected.
   final bool selected;
 
+  /// If non-null, indicates that this subtree represents a button.
```

Comment:
this should probably elaborate on what "a button" means.

Reply:
I'm not sure how to elaborate what a button means, I could try to describe it as something tap-able,  but not everything that's tap-able should have this flag set to true.

Instead I added some documentation of how this affects TalkBack/VoiceOver (provides a "button" hint)